.github/workflows/test-wheel-windows.yml

```diff
@@ -117,11 +117,24 @@ jobs:
         shell: bash --noprofile --norc -xeuo pipefail {0}
         run: ./ci/tools/env-vars test
 
-      - name: Download cuda-pathfinder build artifacts
-        uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093  # v4.3.0
-        with:
-          name: cuda-pathfinder-wheel
-          path: ./cuda_pathfinder
+      - name: Download cuda-pathfinder build artifacts from main branch
```

Comment:
Similarly, we need to move the gh-installation step below to earlier than this new step, otherwise gh is not available.